query/collection_operators.proto

```diff
@@ -202,6 +202,9 @@ message Pagination {
     // The service may impose maximum value.
     // If omitted the service may impose a default value.
     int32 limit = 3;
+    //The bool value enables/disables the record count.
+    //The default setting disables the record count.
+    bool is_total_size = 4;
```

Comment:
How about this ? cc @pnagaraj 
```suggestion
    // The bool value enables/disables the record count.
    // The default setting disables the record count.
    bool is_total_count_required = 4;
```

Reply:
@chinmayb - This is not a required field. 
As comment says it is just to enable or disable the record count.

Reply:
for comment space will modify it

Reply:
the user asks this, hence I was asking that way. It doesn't indicate that its a required field or anything. More like
```
bool is_total_size_needed
```

Reply:
Sure